data/json/items/book/cooking.json

```diff
@@ -255,6 +255,22 @@
     "chapters": 12,
     "copy-from": "book_nonf_hard_cook_tpl"
   },
+  {
+    "id": "cookbook_blood",
+    "type": "BOOK",
+    "name": { "str": "Blood for the Greater Good", "str_pl": "copies of Blood for the Greater Good" },
+    "description": "This recipe collection is compiled by blood-as-food advocate Alžbeta Bátoriová.  Although the consumption of blood is a taboo form of food for Abrahamic religions, but they're old now.",
```

Comment:
"Although the consumption of blood is a taboo form of food for Abrahamic religions, but they're old now." Did you mean to say, "The consumption of blood as a source of food is a taboo in many Abrahamic religions; but that's old world news now." or something like that?

Reply:
It could be "that's a part of the old world" or "that's Old God". I just not want to be specific.